Source/storm/storm.h

```diff
@@ -265,6 +265,9 @@ int getIniInt(const char *keyname, const char *valuename, int defaultValue);
 void setIniInt(const char *keyname, const char *valuename, int value);
 void setIniFloat(const char *keyname, const char *valuename, float value);
 
+bool IniFound(const char *list, const char *item);

```

Comment:
There is no need for these to be public.

Reply:
Hmmm.. if it is not public, inv.cpp, diablo.cpp will get undefined function error.